packages/angular-workspace/nimble-angular/table/nimble-table.directive.ts

```diff
@@ -66,6 +66,14 @@ export class NimbleTableDirective<TData extends TableRecord = TableRecord> imple
         this.renderer.setProperty(this.elementRef.nativeElement, 'selectionMode', value);
     }
 
+    public get actionMenusPreserveSelection(): boolean {
```

Comment:
We have a testing pattern for Angular directives for each of our attributes and properties. You can see it in [nimble-table.directive.spec.ts](https://github.com/ni/nimble/blob/main/packages/angular-workspace/nimble-angular/table/tests/nimble-table.directive.spec.ts)

For each property / attribute we test it in 4 or 5 different configurations (default values, setting via property, setting via attribute, etc). If the table directive doesn't have examples of that pattern for a boolean attribute then you can check what it looks like on another component.